### PR TITLE
This will create a conflict

### PR DESCRIPTION
**Pull request by MarioAriasC**
[x] Ready for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # github-workshop
-
- Mo
+
```
Comment (dosht) [inline README.md]: remove this extra space!!

**Comment by MarioAriasC** [comment]
OK,... boss

File: README.md (hunk 1 above)
Comment (MarioAriasC) [inline README.md]: OK, boss :hankey:

**Comment by dosht** [comment]
:+1:

**Pull request merged**